tests/users/test_dao_change_password.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+from app.api.dao.user import UserDAO
+from app import messages
+from tests.base_test_case import BaseTestCase
+from tests.test_data import test_admin_user
+
+class TestChangePasswordDao(BaseTestCase):
+
+    def test_change_password_dao(self):
+        expected_response = messages.PASSWORD_SUCCESSFULLY_UPDATED, 201
+        actual_response = UserDAO.change_password(
+            user_id = self.admin_user.id,
+            data = {
+                "current_password": test_admin_user["password"],
+                "new_password": "newpassword"
+            }
+        )
+        self.assertEqual(actual_response,expected_response)
```

Comment:
Can we modify this assert to match the assertion done in the previous test file?
```
self.assertEqual(400, actual_response.status_code)
self.assertDictEqual(expected_response,
     json.loads(actual_response.data))
```
So that both your test files are consistent